### PR TITLE
SIDM-3411 Springboot whitelabel error

### PR DESCRIPTION
**Pull request by dfourn**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SIDM-3411

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/error.jsp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                <spring:message code="public.error.page.generic.error" text=""/>
+            </h1>
+        </header>
+        <div class="text">
+            <p class="lede">
+                <spring:message code="public.error.page.generic.sub.error" text=""/>
+            </p>
+        </div>
+    </article>
+    <script>
```
Comment (nikola-naydenov-hmcts) [inline src/main/webapp/WEB-INF/jsp/error.jsp]: Do we need this?

File: src/main/webapp/WEB-INF/jsp/error.jsp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<t:wrapper titleKey="public.error.page.generic.error">
+    <article class="content__body">
+        <header class="page-header group">
+            <h1 class="heading-large">
+                <spring:message code="public.error.page.generic.error" text=""/>
+            </h1>
+        </header>
+        <div class="text">
+            <p class="lede">
+                <spring:message code="public.error.page.generic.sub.error" text=""/>
```
Comment (tbamido) [inline src/main/webapp/WEB-INF/jsp/error.jsp]: does it make any difference having the empty text attribute?

File: src/main/webapp/WEB-INF/jsp/error.jsp (hunk 1 above)
Comment (dfourn) [reply src/main/webapp/WEB-INF/jsp/error.jsp]: It's for the google analytics

File: src/main/webapp/WEB-INF/jsp/error.jsp (hunk 2 above)
Comment (dfourn) [reply src/main/webapp/WEB-INF/jsp/error.jsp]: No difference.

File: src/main/webapp/WEB-INF/jsp/error.jsp (hunk 1 above)
Comment (dfourn) [reply src/main/webapp/WEB-INF/jsp/error.jsp]: I'll remove it

**Pull request merged**